addons/project/static/src/components/project_right_side_panel/project_right_side_panel.js

```diff
@@ -46,6 +46,10 @@ export class ProjectRightSidePanel extends Component {
         onWillStart(() => this.loadData());
     }
 
+    get panelVisible() {
+        return this.state.data.show_milestones || this.state.data.show_project_profitability_helper;
```

Comment:
If ```sale_project``` is installed it ```show_profitability``` depends on billable items to be set . If it is not installed then show profitability is always true.As the case of ```self._show_profitability_helper()``` it is true if ```sale_project``` is installed or else depends on user having a group ```'analytic.group_analytic_accounting'``` it will be set from the users form view. At the first the task was to hide the ```ProjectRightSidePanel``` if there is no data then it went to if the setting is turned on then we should show it independent of data. Also self._show_profitability_helper() giving out false means ```analytic``` has not been installed or setting is turned off.

Ahh then should we only show or hide based on show_profitability_helper()(boolean/setting) , It would be right and same as all the other components in the ```ProjectRightSidePanel``` what do you think?